backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPartnerStatsDAO.java

```diff
@@ -110,9 +110,9 @@ public BigDecimal retrieveOpenItems(@NonNull final BPartnerStats bpStats)
 
 		BigDecimal openItems = null;
 
-		final Object[] sqlParams = new Object[] { stats.getC_BPartner_ID() };
+		final Object[] sqlParams = new Object[] { SystemTime.asDate(), stats.getC_BPartner_ID() };
 		final String sql = "SELECT "
-				+ "currencyBase(openamt,C_Currency_ID,DateInvoiced,AD_Client_ID,AD_Org_ID) from de_metas_endcustomer_fresh_reports.OpenItems_Report(now()::date) where  C_BPartner_ID=?";
+				+ "COALESCE((SELECT SUM(currencyBase(openamt,C_Currency_ID,DateInvoiced,AD_Client_ID,AD_Org_ID)) from de_metas_endcustomer_fresh_reports.OpenItems_Report(?, ?)), 0)";
```

Comment:
not sure this will work since the function has 3 params, and you want to specify only the first and the third one.

i suggest to use named params to be on the safe side.

Reply:
it will, but I agree, it is not really suggestive, that's how I did it here https://github.com/metasfresh/metasfresh/pull/19947/commits/76a48c053c89117e84a241428b0b9ec270495ea2, is it ok like that?

Reply:
thx